stylesheets/index.css

```diff
@@ -12,3 +12,100 @@
     outline: 1px solid transparent;
     border: none;
 }
+
+.img-container {
+    display: inline-flex;
+}
+
+.img-relative {
+    position: relative;
+    width: 50%;
+    height: 100%;
+}
+
+.crypto-flex {
+    display: grid;
+    grid-template-columns: repeat(2,2fr);
+    align-items: center;
+    height: calc(100vh - 53px);
+}
+
+.btn {
+    align-self: center;
+    border: none;
+}
+
+.category-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.category {
+    border: 0.1rem solid var(--lynx-grey-color);
+    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+}
+
+.category:hover {
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+}
+
+.content {
+    width: 60%;
+    margin: auto;
+}
+
+.content-fluid {
+    width: 50%;
+    margin: auto;
+}
+
```

Comment:
both are having same properties so you could have a class name margin-auto and used that in both of them

Reply:
done